src/Products/urban/migration/update_250.py

```diff
@@ -65,3 +65,21 @@ def replace_mailing_loop_owners(context):
                 new_value.append(line)
             template.context_variables = new_value
     logger.info("upgrade done!")
+
+
+def fix_type_eventtype_in_config(context):
+    """
+    Sometimes the type of the eventtype in the config is a string instead of what is expected.
+    """
+    logger= logging.getLogger('urban: fix type of eventtype in config')
+    logger.info("starting upgrade steps")
+    config = api.portal.get_tool('portal_urban')
+    all_eventconfigs = []
+    for licenceconf in config.get_all_licence_configs():
+        all_eventconfigs.extend(licenceconf.getEventConfigs())
+    for eventc in all_eventconfigs:
+        eventtype = eventc.getEventType()
+        if isinstance(eventtype, basestring):
+            eventc.eventType = [eventtype]
+            logger.info("type modification test: is eventType a string: {} ").format(isinstance(eventc.eventType, basestring))
```

Comment:
Je loggerais plutot des infos de l'objet corrigé pour qu'on voie ce qui a été corrigé
sinon ça va juste afficher plusieurs fois "type modification test: is eventType a string: True"
ce qui n'est pas super informatif :p..